docs/releases/v13/upgrade-v13.0.md

```diff
@@ -84,11 +84,11 @@ Update the `<my-site>/Pipfile` by changing the `version` of `invenio-app-rdm` to
 +++invenio-app-rdm = {extras = [...], version = "~=13.0.0"}
 ```
 
-If you're using [Sentry](https://sentry.io) (tool for monitoring or error tracking), update invenio-logging in `<my-site>/Pipfile` to
+If you're using [Sentry](https://sentry.io) (tool for monitoring or error tracking), update the dependency in `<my-site>/Pipfile` to:
 
 ```diff
 ---invenio-logging = {extras = ["sentry_sdk"], version = "~=2.0"}
-+++invenio-logging = {extras = ["sentry"]}
++++sentry-sdk = {extras = ["flask"], version = ">=1.0.0,<2.0.0"}
```

Comment:
Why does the previous `extras` not work given: https://github.com/inveniosoftware/invenio-logging/blob/master/setup.cfg#L44 ?

Reply:
There was no pinning of [invenio-logging](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/setup.cfg#L42). I think that it is a safer bet to only install `sentry`, without hardcoding the invenio-logging version there: it will not allow us to release major versions without having to write another doc to make sure that the range is updated.